CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log - AWS AppSync SDK for Android
 
+## [Release 2.10.1](https://github.com/awslabs/aws-mobile-appsync-sdk-android/releases/tag/release_v2.10.1)
+
+### Bug Fixes
+* Fixed Uncaught "Call is cancelled" subscription exception See [issue#202](https://github.com/awslabs/aws-mobile-appsync-sdk-android/issues/202)
```

Comment:
can you add more details like:

The callback passed in will be invoked when the subscriptions are cancelled, failed and in any other scenarios.